<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI::OssSettingsWidget</class>
 <widget class="QWidget" name="OssOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>79</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="nameGroup">
     <property name="title">
      <string>OSS settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDevice">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMixer">
        <property name="text">
         <string>Mixer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mixer"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="device"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="selectDevice">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="selectMixer">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>device</tabstop>
  <tabstop>selectDevice</tabstop>
  <tabstop>mixer</tabstop>
  <tabstop>selectMixer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
